gulpfile.js

```diff
@@ -24,16 +24,21 @@ var argv = yargs
   .command('build', 'generate optimimized third party library for production',
     function(yargs) {
       argv = yargs
-        .usage('Usage: $0 build [--minify]')
+        .usage('Usage: $0 build [--minify] [--output_directory]')
         .option('minify', {
           describe: 'Whether to minify third-party dependencies'
-        }).argv;
+        })
+        .option('output_directory', {
+          describe: 'A path to the directory where the files will be generated'
+        })
+        .argv;
```

Comment:
I agree with @BenHenning for readability, it will make more sense if it stays at the line above rather than leaving `})` hanging.